src/main/java/com/snowflake/kafka/connector/internal/OAuthConstants.java

```diff
@@ -0,0 +1,21 @@
+package com.snowflake.kafka.connector.internal;
+
+/**
+ * This class contains constants for OAuth request.
+ *
+ * @see <a
+ *     href="https://github.com/snowflakedb/snowflake/blob/4fdb96cd5849f266cda430c5d49a13c29e866af5/GlobalServices/src/main/java/com/snowflake/resources/ResourceConstants.java">ResourceConstants</a>
+ */
+public class OAuthConstants {
+  public static final String TOKEN_REQUEST_ENDPOINT = "/oauth/token-request";
+  public static final String OAUTH_CONTENT_TYPE_HEADER = "application/x-www-form-urlencoded";
+  public static final String BASIC_AUTH_HEADER_PREFIX = "Basic ";
+  public static final String GRANT_TYPE_PARAM = "grant_type";
+  public static final String REFRESH_TOKEN = "refresh_token";
+  public static final String ACCESS_TOKEN = "access_token";
+  public static final String SNOWFLAKE_JWT = "snowflake_jwt";
```

Comment:
nit: jwt constant shouldnt be part of this

Reply:
Fixed.